src/app.js

```diff
@@ -102,6 +110,11 @@ app.post("/api/movies/review", (request, response) => {
   postRequest(url, jsonData, response);
 });
 
+app.get("/api/movies/rating/:id", async (req, res) => {
+  const data = await averageRating(req.params.id);
+  res.send(data);
+});
+
```

Comment:
If you like you can add some catch(error) etc like in the rest of the routes.

Reply:
Added try, catch to route and to function averageRating in movie.js-file